prepline_general/api/general.py

```diff
@@ -218,6 +237,14 @@ def pipeline_api(
     show_coordinates_str = (m_coordinates[0] if len(m_coordinates) else "false").lower()
     show_coordinates = show_coordinates_str == "true"
 
+    hi_res_model_name = m_hi_res_model_name[0] if len(m_hi_res_model_name) else None
+
+    if hi_res_model_name and hi_res_model_name in CHIPPER_MODEL_TYPES and show_coordinates:
+        raise HTTPException(
+            status_code=403,
```

Comment:
I think an HTTP 400 would be more suitable than an HTTP 403 because they are requesting an unsupported action rather than an action that specifically their account isn't authorized for.